tests/stress/consume_flow_with_threads_test.py

```diff
@@ -157,8 +134,43 @@ def concurrent_consume_flow(concurrent_flows: int, repetitions: int):
 
 
 @pytest.mark.slow
-@pytest.mark.parametrize(
-    ["concurrent_flows", "repetitions"], [(1, 300), (3, 100), (20, 5)]
-)
+@pytest.mark.parametrize(["concurrent_flows", "repetitions"], [(1, 2), (3, 1), (5, 5)])
 def test_concurrent_consume_flow(concurrent_flows, repetitions):
     concurrent_consume_flow(concurrent_flows, repetitions)
+
+
+def _create_downloads_path(config):
+    destination = config.downloads_path
+    if not os.path.isabs(destination):
+        destination = os.path.abspath(destination)
+
+    if os.path.exists(destination) and len(os.listdir(destination)) > 0:
+        list(
+            map(
+                lambda d: shutil.rmtree(os.path.join(destination, d)),
+                os.listdir(destination),
+            )
+        )
+    if not os.path.exists(destination):
+        os.mkdir(destination)
+
+    assert len(os.listdir(destination)) == 0
+
+    return destination
+
+
+def _create_files():
+    file1_url = "https://raw.githubusercontent.com/tbertinmahieux/MSongsDB/master/Tasks_Demos/CoverSongs/shs_dataset_test.txt"
```

Comment:
let's create a new issue for file fixtures :)
I think to avoid duplication we can keep these definitions inside the readme tests, but I've seen the tbertinmahieux/MSongsDB file so many times in our test suite that I'm pretty sure it warrants a fixture.

Reply:
Sure, I'll create an issue that creates fixtures for these files and paste it here.

Reply:
I believe the PR for adding file fixtures can be found here: https://github.com/oceanprotocol/ocean.py/pull/792